interview-questions.md

```diff
@@ -8,50 +8,51 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is the difference between a parameter and an argument?
 
-Your answer:
+Your answer: A parameter is a variable, value, or information in a function, and arguement is the actually data that gets passed through the function. 
 
-Researched answer:
+Researched answer: A Parameter is a placeholder for the data being passed through the function and an arguement is the data that actually gets passed into the function. I like to think of it as a parameter is a reference for a reference meaning we are taking in all the necessary information in one variable to be invoked upon in the function. An arguement is the actual information. 
 
 2. The JavaScript built in method .map() takes predefined parameters. What are they? Which are required and which are optional?
 
-Your answer:
+Your answer: The built in method.map will take an individual data set, an array. It takes the same 3 arguements like .forEach. It was always return as a new array, of the same length as the original array. It will do something about each item in the item. A return is required when using a the .map method. 
 
-Researched answer:
+Researched answer: ____________
```

Comment:
I don't see a researched answer here.  Keep in mind, there is always something you can extend on your knowledge when researching.